### ISSUE TITLE
Web Accessibility Checklist inaccurately suggests adding roles to elements where role is already default

### ISSUE THREAD

**Issue by gilluminate** [a11yproject/a11yproject.com#354]
HTML5 elements come with default roles and the W3 specs explicitly say not to set them to those defaults.

For example, looking at the [spec for the `<header>` element](http://www.w3.org/TR/html5/sections.html#the-header-element) you will notice within the section titled **Allowed ARIA role attribute values** it is indicated that *banner* and *presentation* are the two allowed roles for that element and in parentheses for the banner role shows that it is the default role and emphasis is given to the note **"do not add"** which links to a comprehensive list of default roles.

In a nutshell, the only item in the **Landmarks** section which *should* contain a role attribute is the example for adding the search role to a form element that is to be used as the main search.

**Comment by joe-watkins** [comment]
@gilluminate This is a highly debated topic for sure :) Not all Assistive Technology handles these implied roles the same,.. so to be safe we can add them to the markup without causing harm.

**Comment by gilluminate** [comment]
Ah, got it. I didn't realize that was the case. Thanks for the clarification, @joe-watkins!

**Comment by joe-watkins** [comment]
@gilluminate no prob :) tnx for contributing!
